homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -134,7 +136,7 @@
 SERVICE_STANDBY = "standby"
 
 # pylint: disable=unnecessary-lambda
-DEVICE_SCHEMA = vol.Schema(
+DEVICE_SCHEMA: vol.Schema = vol.Schema(
```

Comment:
Is this necessary? I would think the type checker could implicitly understand this constant's type.

Reply:
Yes it is nessecary, otherwise mypy complaind.

Reply:
Remark the lambda is the problem, normally it is not needed.

Reply:
When I remove it I get:
homeassistant/components/hdmi_cec/__init__.py:142: error: Cannot determine type of "DEVICE_SCHEMA"  [has-type]
Which is due to the lambda in the schema.
Remark I removed "pylint: disable=unnecessary-lambda" as you can see, because that would not satisfy mypy, but ignore satisfies both.